### PR TITLE
Maven publishing.

### PR DESCRIPTION
**Pull request by xd720p**
Add working publication script.

These changes allow to publish library into bintray by calling `bintrayUpload` task. But PhotoView dependency requires jitpack, so library's user must add this repository in his project.

### CODE REVIEW

File: gallery/build.gradle [hunk 1]
```diff
@@ -1,11 +1,10 @@
 plugins {
```
Comment (terrakok) [inline gallery/build.gradle]: I think we can delete bintray plugin

File: gallery/build.gradle [hunk 2]
```diff
@@ -85,3 +48,4 @@ dependencies {
     // Image loader
     implementation 'com.github.bumptech.glide:glide:4.7.1'
 }
+apply from: 'publishing.gradle'
```
Comment (terrakok) [inline gallery/build.gradle]: Why in the end of file?

File: gallery/build.gradle (hunk 1 above)
Comment (xd720p) [reply gallery/build.gradle]: But we can't publish library on bintray without bintray plugin.

File: gallery/build.gradle (hunk 2 above)
Comment (xd720p) [reply gallery/build.gradle]: This `apply` should be after `android` block to get compileSdkVersion, so i decided to move it in the end of file.

File: gallery/build.gradle (hunk 1 above)
Comment (xd720p) [reply gallery/build.gradle]: Deleted this piece of code and replcaed it with `classpath` block in project gradle scrypt and `apply plugin` in `publishing.gradle`

File: gallery/build.gradle [hunk 3]
```diff
@@ -26,49 +22,15 @@ android {
     }
 }
 
+apply from: 'publishing.gradle'
```
Comment (terrakok) [inline gallery/build.gradle:25]: All `apply` move to top :)

**Review by terrakok — APPROVED** [review]

**Pull request merged**